### PR TITLE
Adding prod ES to the CI

### PR DESCRIPTION
**Pull request by vishnuchalla**
## Type of change

- [x] Refactor
- [ ] New feature
- [ ] Bug fix
- [x] Optimization
- [ ] Documentation Update

## Description
Removing publicly exposed dev ES instance. Will delete this experimental branch once this PR is merged.

## Related Tickets & Documents
- Closes # https://issues.redhat.com/browse/PERFSCALE-3430

## Testing
Can only be tested once this PR is merged into `main` branch

### CODE REVIEW

**Comment by vishnuchalla** [comment]
Needs this PR to be merged in order for the CI tests to go through. More context [here](https://github.com/kube-burner/kube-burner-ocp/pull/116#discussion_r1790904368). Verified the workflow in fork [repo](https://github.com/vishnuchalla/kube-burner/actions/runs/11222702170)

File: .github/workflows/test-k8s.yml [hunk 1]
```diff
@@ -1,14 +1,8 @@
...
 on:
+  workflow_dispatch:
   workflow_call:
-    secrets:
-      OPENSHIFT_SERVER:
-        required: true
-      OPENSHIFT_USER:
-        required: true
-      OPENSHIFT_PASSWORD:
-        required: true
```
Comment (rsevilla87) [inline .github/workflows/test-k8s.yml:11]: I don't recall now why I had to do add these secrets here... I assume you tested it after removing them, correct?

Comment (vishnuchalla) [reply .github/workflows/test-k8s.yml:11]: These secrets are not required in the core repo, as we do not use a openshift cluster at all while running CI tests.

**Review by rsevilla87 — APPROVED** [review]
lgtm

**Pull request merged**